pytorch_lightning/core/hooks.py

```diff
@@ -20,7 +20,7 @@
 
 from pytorch_lightning.utilities import move_data_to_device
 from pytorch_lightning.utilities.types import EVAL_DATALOADERS, STEP_OUTPUT, TRAIN_DATALOADERS
-from pytorch_lightning.utilities.warnings import rank_zero_deprecation, rank_zero_warn
```

Comment:
why this change? :)

Reply:
the linter in the `pre-commit.ci` caught an unused import there 😅